### DISCUSSION TITLE
ADDING BACKGROUND IMAGE TO STATIC HOSTED PAGE

### DISCUSSION THREAD

**Discussion by thrila** [tailwindlabs/tailwindcss#8912] [Help]
I added a background image in my project and  it worked on the live version i then ran `npm build && npm export ` and made the site live on github when the site became live the rest of my CSS worked but my image wouldn't work 

the way i imported inserted the image is `background-image: url('../public/image 2.PNG'); `

**Comment by tofuwu-dev** [comment]
Do you use Next.js?
have you tried to write like this?
```typescript
background-image: url('/image2.png');
```

**Reply by thrila** [reply]
yes it worked when my project wasnt live when  i made it static and hosted it on github i added akamai image loader and tweaked the image path a bit and it worked but the background image wouldnt work any ideas?

**Reply by tofuwu-dev** [reply]
If you use Next,js, could you plz qoute here your next config file?

**Reply by tofuwu-dev** [reply]
I suppose, you image assets are in your local, right?
